src/constants/buckets.js

```diff
@@ -0,0 +1 @@
+export const AWS_STAGING_BUCKET = "staging.interactives.politico.com";
```

Comment:
So you still need this for the output share, but let's make another one where we start putting the template which is `artisan.politico.com`.